src/app/table-transactions/table-transactions.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from "@angular/core";
+import { DataSource } from "@angular/cdk/table";
+
+@Component({
+  selector: "app-table-transactions",
+  templateUrl: "./table-transactions.component.html",
+  styleUrls: ["./table-transactions.component.scss"]
+})
+export class TableTransactionsComponent {
```

Comment:
you do not need to have `component` part in the name -- just `TableTransactions`

Reply:
why not? it is automatically generated name by Angular CLI

Reply:
ok